sympy/plotting/plot.py

```diff
@@ -1213,32 +1217,32 @@ class BaseBackend:
     How the plotting module works:
 
     1. Whenever a plotting function is called, the provided expressions are
-        processed and a list of instances of the `BaseSeries` class is created,
-        containing the necessary information to plot the expressions (eg the
-        expression, ranges, series name, ...). Eventually, these objects will
-        generate the numerical data to be plotted.
-    2. A Plot object is instantiated, which stores the list of series and the
+        processed and a list of instances of the :class:`BaseSeries` class is
+        created, containing the necessary information to plot the expressions
+        (e.g. the expression, ranges, series name, ...). Eventually, these
+        objects will generate the numerical data to be plotted.
+    2. A :class:`~.Plot` object is instantiated, which stores the list of series and the
```

Comment:
Is this line now too long or does `:class:~` not count for the length?

Reply:
I do not think we enforce line length in SymPy and since this is text it should wrap anyway. 

But I probably missed re-wrapping the source anyway.

Reply:
I don't think it is forced either, but since I saw you had rewrapped it in other places, I assumed it was intended to be so.